integrations/airflow/marquez_airflow/extractors/sql/parser.py

```diff
@@ -85,19 +104,57 @@ def parse(sql: str) -> SqlMeta:
         # We assume only one statement in SQL
         tokens = TokenList(statements[0].tokens)
```

Comment:
It looks like we could easily iterate on the statements if there's more than one and return all inputs and outputs

Reply:
Maybe in following PR? I think returning those would require more changes in parser and code that uses parser, since I don't want to just union input and output datasets, they should be separate.